html/janus.js

```diff
@@ -230,8 +232,8 @@ Janus.init = function(options) {
 		// Already initialized
 		options.callback();
 	} else {
-		if(typeof console == "undefined" || typeof console.log == "undefined") {
-			console = { log: function() {} };
+		if(typeof console.log == "undefined") {
```

Comment:
Are you 100% sure `console` is always defined?

Reply:
Yes, `console.log` should always be defined (so we could potentially remove the entire block). https://developer.mozilla.org/en-US/docs/Web/API/Console/log#browser_compatibility

Reply:
Again, this is not only used in browsers though.

Reply:
@davel I think you should revert this change as it is not related to the linter at all

Reply:
The assignment to `console` on line 234 is picked up as an error. Would you prefer me to remove the entire block, it should never be executed?

The `console` object should also exist as a global variable in Node.js too, https://nodejs.org/api/console.html

Reply:
I see thanks for clarifying.
I guess we can leave as it is in the PR then.